docs/user/index.rst

```diff
@@ -92,126 +92,76 @@ Open Source and User Focused |:heartbeat:|
+Reference
+---------
 
-* **Topic specific guides**:
-  :doc:`How-to guides <guides/index>`
+📚 :doc:`/reference/features`
+  An overview and reference of all features of Read the Docs.
 
-* **Extending Read the Docs**:
-  :doc:`REST API <api/index>`
+📚 :doc:`/config-file/index`
+  Reference for the ``.readthedocs.yaml`` file.
 
+📚 :doc:`/builds`
+  Reference to how builds are processed
 
+📚 :doc:`/build-customization`
+  Reference of the ways that you can add your own build logic or replace the default build steps.
 
-Read the Docs for Business
---------------------------
+📚 :doc:`/server-side-search/syntax`
+  How our search works - available for all documentation projects.
 
-Read the Docs has a commercial offering with improved support and additional features.
+📚 :doc:`/api/index`
+  Building a documentation project on Read the Docs gives access to a number of great APIs that you can take advantage of on your website and integration tools.
 
-* **Read the Docs for Business**:
-  :doc:`Organizations <commercial/organizations>` |
-  :doc:`Single sign on <commercial/single-sign-on>` |
-  :doc:`Project privacy level <commercial/privacy-level>` |
-  :doc:`Sharing externally <commercial/sharing>`
+📚 `Developer documentation <https://dev.readthedocs.io/>`__
+  Do you want to help develop Read the Docs? This is where to get started.
 
 
-The Read the Docs project and organization
-------------------------------------------
-
-Learn about Read the Docs, the project and the company,
-and find out how you can get involved and contribute to the development and success
-of Read the Docs and the larger software documentation ecosystem.
-
-* **Policies & Process**:
-  :doc:`security` |
-  :doc:`DMCA takedown policy <dmca/index>` |
-  :doc:`/abandoned-projects` |
-  :doc:`/unofficial-projects` |
-  :doc:`Release notes & changelog <changelog>`
-
+Explanation
+-----------
 
-* **The people and philosophy behind Read the Docs**:
-  :doc:`About us </about/index>` |
-  :doc:`Team <team>` |
-  :doc:`Open source philosophy <open-source-philosophy>` |
-  :doc:`Our story <story>`
+.. TODO: This next item needs its article to be finished in a separate PR
+.. https://github.com/readthedocs/readthedocs.org/pull/10071
 
-* **Financial and material support**:
-  :doc:`advertising/index` |
-  :doc:`Sponsors <sponsors>`
+💡 :doc:`Continuous Documentation </integrations>`
+  Learn about the many advantages of having your documentation continuously build and deployed.
 
-* **Legal documents**:
-  :doc:`Terms of service <terms-of-service>` |
-  :doc:`Privacy policy <privacy-policy>` |
-  :doc:`Data processing agreement <legal/dpa/index>`
+.. TODO: This next item needs its article to be finished in a separate PR
+.. https://github.com/readthedocs/readthedocs.org/pull/10071
 
-* **Getting involved with Read the Docs**:
-  :doc:`/glossary` |
-  :doc:`Developer documentation <rtd-dev:index>`
+💡 Advanced topics: Deep-dive into Read the Docs
```

Comment:
I think we can probably just link to https://docs.readthedocs.io/en/stable/choosing-a-site.html for now, and ship this PR, and then add the index updates in the dependent PR?